packages/server/tests/acceptance/serverConfig.spec.ts

```diff
@@ -17,16 +17,18 @@
  * limitations under the License.
  *
  */
+import { ConfigService } from '@hashgraph/json-rpc-config-service/dist/services';
 import { expect } from 'chai';
+
+import { ConfigName } from '../../../config-service/src/services/configName';
 import { Utils } from '../helpers/utils';
-import { ConfigService } from '@hashgraph/json-rpc-config-service/dist/services';
 
 describe('@server-config Server Configuration Options Coverage', function () {
   describe('Koa Server Timeout', () => {
     it('should timeout a request after the specified time', async () => {
-      const requestTimeoutMs: number = parseInt(ConfigService.get('SERVER_REQUEST_TIMEOUT_MS') || '3000');
+      const requestTimeoutMs: number = parseInt(ConfigService.get(ConfigName.SERVER_REQUEST_TIMEOUT_MS) as string || '3000');
```

Comment:
I wonder if we should add a getString and do the casting in the ConfigService to make this easier on devs.
Thinking out loud and getting thoughts.
No need to change in this PR

Reply:
Thank you for the suggestion about `ConfigService.getString`. I agree that it could streamline type handling. I suggest we open a separate issue to implement and test this change comprehensively.